package.json

```diff
@@ -45,24 +45,28 @@
     "@salesforce/ts-types": "^1.0.0",
     "@types/graceful-fs": "^4.1.3",
     "@types/jsforce": "1.9.20",
+
+    "axios": "^0.19.2",
```

Comment:
Doesn't jsforce already make calls to `services/oauth2/token`? Why do we need to bring in another library to do this?